modules/tensor_mechanics/include/materials/ADComputeShellStress.h

```diff
@@ -47,8 +47,8 @@ class ADComputeShellStress : public Material
   /// Quadrature points along thickness
   std::vector<Point> _t_points;
 
-  /// Rotation matrix material property
-  std::vector<const MaterialProperty<RankTwoTensor> *> _rotation_matrix;
+  /// Rotation matrix  material property with covariant base vector terms
```

Comment:
Extra space before 'material'

Reply:
Since Ben already chimed in... You could change the documentation of these member variables to reflect they are covariant/contravariant transformation matrices (not just rotation matrices)